### PR TITLE
[MIG][10.0] website_openstreetmap module

### PR DESCRIPTION
**Pull request by mFlayyan**

### CODE REVIEW

File: website_openstreetmap/models/res_company.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            'mlon': self.partner_longitude,
+            'zoom': zoom,
+            'layers': 'M',
+            'map': "{zm}/{lat}/{lon}".format(
+                zm=zoom,
+                lat=self.partner_latitude or '',
+                lon=self.partner_longitude or '', )
+        }
+        return url + '?%s' % urllib.urlencode(params)
+    """
```
Comment (tarteo) [inline website_openstreetmap/models/res_company.py:58]: Is this still needed or can it be removed?

File: website_openstreetmap/models/res_company.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+#
+##############################################################################
+from openerp import models, fields, api, exceptions, _
+import urllib
+import json
+
+
+class ResCompany(models.Model):
+
+    _inherit = "res.partner"
```
Comment (tarteo) [inline website_openstreetmap/models/res_company.py:27]: This should be res.company right?

File: website_openstreetmap/models/res_company.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+    def openstreetmap_img(self,  zoom=16, width=250, height=250):
+        url = 'http://staticmap.openstreetmap.de/staticmap.php'
+        params = {
+            'center': "{lat},{lon}".format(
+                lat=self.partner_latitude or '',
```
Comment (tarteo) [inline website_openstreetmap/models/res_company.py:33]: If this is model is res.company then this should be self.partner_id.partner_latitude

File: website_openstreetmap/views/widget.xml [hunk 4]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+
+        <!-- widget structure inferred from image floating widget it is placed in content and can be inside a text -->
+        <template id="openstreetmap_widget" inherit_id="website.snippets" name="openstreetmap">
+            <xpath expr="//div[@id='snippet_content']" position="inside"> <!--this decides the category the snippet belongs to -->
+                <div name="openstreetmap_floating" class="o_block_image_float">
```
Comment (tarteo) [inline website_openstreetmap/views/widget.xml:8]: The standard way to define a snippet is like this:

`<t t-snippet="website.s_well" t-thumbnail="/website/static/src/img/blocks/block_well.png"/>`

`<template id="s_well" name="Well"></template>`

**Review by tarteo — CHANGES_REQUESTED** [review]

**Comment by simahawk** [comment]
@mFlayyan still working on this?
I'm closing this as it's 1yr old and there's no answer to review.
Feel free to open it again if needed.
Thanks!

**Pull request closed without merging**